tests/test_habitat_quality.py

```diff
@@ -2045,12 +2044,10 @@ def test_habitat_quality_argspec_missing_threat_header(self):
                 '0.07,1.0,threat_2,threat_2_c.tif,threat_2_f.tif\n')
 
         validate_result = habitat_quality.validate(args, limit_to=None)
-        self.assertTrue(
-            validate_result,
-            "expected failed validations instead didn't get any.")
-        self.assertTrue(
-            "Fields are missing from this table: ['DECAY']" in
-            validate_result[0][1], validate_result[0][1])
+        expected = [(
+            ['threats_table_path'],
+            'decay matched 0 headers, expected at least one')]
```

Comment:
Similar comment to GLOBIO here. I think we should try to rephrase this to something like "expected columns ['decay', ... ] in the table but did not find 'decay'."

Reply:
see https://github.com/natcap/invest/pull/526#discussion_r652766326